docs/sources/setup-grafana/configure-docker.md

```diff
@@ -103,15 +103,15 @@ You can install publicly available plugins and plugins that are private or used
 
 ### Install plugins from other sources
 
-To install plugins from other sources, you must define the custom URL and specify it immediately before the plugin name in the `GF_INSTALL_PLUGINS` environment variable: `GF_INSTALL_PLUGINS=<url to plugin zip>;<plugin name>`.
+To install plugins from other sources, you must define the custom URL and specify it immediately before the plugin name in the `GF_PLUGINS_PREINSTALL` environment variable: `GF_PLUGINS_PREINSTALL=<plugin ID>@[<plugin version>]@<url to plugin zip>`.
```

Comment:
I am not sure about this change since it will only work with Grafana versions 11.5. Should I change this or keep the legacy way with `GF_INSTALL_PLUGINS`?